dev_docs/01_getting_started.md

```diff
@@ -4,8 +4,14 @@ The design system has two primary parts, the UI component library [`/lib`](../li
 
 ## Prerequisites
 
-- Node.js 18.x (see [Node Version Manager](https://github.com/nvm-sh/nvm))
-- Yarn >=1.22.22
+To ensure consistency across development environments, we use Volta to manage Node.js and Yarn versions.
+
+**Steps**:
+
+1. Install Volta (if not already installed):
+   ```bash
+   curl https://get.volta.sh | bash
+   ```
```

Comment:
We probably should not provide the actual steps to install Volta itself, but rather, we should link to the official documentation. While I do get that this might be more convenient for the user; however, it can cause issues if the official documentation moves to a different installation method or introduces any changes. 

Plus this command won't work for the users on Windows as they have Powershell instead of bash. Letting the official documentation handle the OS & platform specific requirements is always a good choice IMHO